jablib/build.gradle.kts

```diff
@@ -70,7 +81,17 @@ dependencies {
     implementation ("org.apache.pdfbox:fontbox")
     implementation ("org.apache.pdfbox:xmpbox")
 
-    implementation("net.sourceforge.tess4j:tess4j")
+    implementation("net.sourceforge.tess4j:tess4j") {
+        exclude(group = "net.java.dev.jna", module = "jna")
+        exclude(group = "net.java.dev.jna", module = "jna-platform")
+    }
+
+    implementation(
+        "net.java.dev.jna:jna:${jnaVersion}"
```

Comment:
Put concrete versions at https://github.com/JabRef/jabref/blob/main/versions/build.gradle.kts

They should be taken - no exclusions needed IMHO